values.yaml

```diff
@@ -127,3 +127,35 @@ dashboard:
 chaosDaemon:
   runtime: containerd
   socketPath: /run/containerd/containerd.sock
+
+fleetscheduler:
+  enable: false # when enabled, metal-toolbox/fleet-scheduler will need to be deployed with `make push-image-devel`
+  image:
+    repository: "localhost:5001/fleet-scheduler"
+    tag: latest
+    pullPolicy: Always
+  env:
+    log_level: debug
+    facility: fr9
```

Comment:
where is this `facility` value used in the scheduler?

Reply:
when calling serverserviceClient.List(ctx, params)
https://github.com/metal-toolbox/hollow-serverservice/blob/9089164ea19a188d3e5eb79d56892e23d9b2f121/pkg/api/v1/server_service.go#L100

params is a ServerListParams
which needs a facility

https://github.com/metal-toolbox/hollow-serverservice/blob/9089164ea19a188d3e5eb79d56892e23d9b2f121/pkg/api/v1/server_list_params.go#L14

Reply:
ah righto, can you then set the value to `sandbox`, and also update the README which is incorrect at the moment - https://github.com/metal-toolbox/sandbox/blob/main/README.md?plain=1#L57

Reply:
ah perfect. Yea i was using that value. Ill change it to "sandbox"